src/iomusxml.cpp

```diff
@@ -2852,8 +2852,15 @@ void MusicXmlInput::ReadMusicXmlNote(
                 Artic *artic = new Artic();
                 artics.push_back(ConvertArticulations(articulation.name()));
                 if (!std::strcmp(articulation.name(), "detached-legato")) {
+                    // we need to split up this one
                     artics.clear();
                     artics.push_back(ARTICULATION_stacc);
+                    artic->SetArtic(artics);
+                    artic->SetColor(articulation.attribute("color").as_string());
+                    artic->SetPlace(artic->AttPlacement::StrToStaffrel(articulation.attribute("placement").as_string()));
+                    element->AddChild(artic);
+                    artics.clear();
+                    artic = new Artic();
```

Comment:
You also need to set the color and the place to that one

Reply:
It is done 8 lines below.

Reply:
Ah yes, sorry, I overlooked it. I must say that the logic is now a bit cumbersome. It is a bit weird to add the value to `artics` (line 2853) and only after to check what it is and to remove it if it is a 'stacc ten'.

Reply:
Tried to improve it a bit.